auth.go

```diff
@@ -95,6 +102,16 @@ func New(opts Options) (Service, error) {
 		return nil, err
 	}
 
+	buttonTpl, err := readTemplate(opts.ButtonTpl)
```

Comment:
this returns an error if the template does not exist, which will happen if the path is `""` (when you don't want it), only should be loaded if `opts.ButtonTpl != ""`